File: solucoes/python/025.py
#25. Elabore um algoritmo que lê 10 números e imprime a metade e o quadrado e o cubo de cada número.

for x in range(10):
    numero = int(input())
    metade = numero/2
    quadrado= numero**2
    cubo = numero**3
    print(metade,quadrado,cubo)

File: solucoes/dart/025.dart
// 25. Elabore um algoritmo que lê 10 números e imprime a metade e o quadrado e o cubo de cada número.

import 'dart:io';
import 'dart:math';
main(){
  for (int i=0; i<10; i++){
    double numero = double.parse(stdin.readLineSync());
    double metade = numero/2;
    double quadrado = pow(numero, 2);
    double cubo = pow(numero, 3);
    print("metade: $metade, quadrado: $quadrado, cubo: $cubo");
  }
}